test/living-html/location.js

```diff
@@ -17,7 +17,9 @@ exports["window.location.port for an about:blank window"] = function (t) {
 };
 
 exports["window.location.hash is manipulable"] = function (t) {
-  var window = jsdom.jsdom().parentWindow;
+  var window = jsdom.jsdom("", {
```

Comment:
Changing hash specifically works with the patch, however changing search for example still breaks so I kinda changed for uniformity.